components/tabs/tabs.js

```diff
@@ -1138,6 +1202,9 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 
 		await this.updateComplete;
 
+		selectedTab.selected = true;
```

Comment:
I had fixed this [here](https://github.com/BrightspaceUI/core/pull/5462/files#r2010127406) but I've pulled it into here so that the demo for this PR works properly